Tests/LibC/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ set(TEST_SOURCES
     TestLibCSetjmp.cpp
     TestLibCString.cpp
     TestLibCTime.cpp
+    TestLibPthreadCleanup.cpp
+    TestLibPthreadSpinLocks.cpp
+    TestLibPthreadRWLocks.cpp
```

Comment:
Why are these still called `LibPthread`?

Reply:
I remember renaming them, so presumably that change went missing while I was redoing all my changes to split them up some more. Should be fixed now! :^)